src/styles/modules/nav-list-iteam-text.scss

```diff
@@ -0,0 +1,19 @@
+.nav-list-item { 
+   margin: 0 58px 0 58px;
+   padding: 34px 0 34px 0;
+}
+
+li a:hover {
```

Comment:
your module could have base tags as selectors but not as a root, example: `.nav-list-item li`, but in your case you don't need that because you have css classes for both `li` and `a`, please use them instead of plain tags as selectors

Reply:
done